blueocean-dashboard/src/main/js/components/RunPipeline.jsx

```diff
@@ -0,0 +1,99 @@
+/**
+ * Simple widget for triggering the running of a pipeline.
+ */
+
+import React, { Component, PropTypes } from 'react';
+import Pipeline from '../api/Pipeline';
+import Branch from '../api/Branch';
+import { Toast } from '@jenkins-cd/design-language';
+
+export default class RunPipeline extends Component {
+
+    constructor(props) {
+        super(props);
+        const pipeline = new Pipeline(props.organization, props.pipeline);
+        this.branch = new Branch(pipeline, props.branch);
+        this.state = {
+            toast: undefined,
+        };
```

Comment:
Small style thing but I usually prefer to initialize "empty" values to `null` so I can differentiate between it and a value which is truly undefined.